shared/src/commonMain/kotlin/org/hyperskill/app/main/presentation/AppFeature.kt

```diff
@@ -19,11 +19,14 @@ interface AppFeature {
         object UserAccountStatusError : Message
         object OpenAuthScreen : Message
         object OpenNewUserScreen : Message
+        object AppLaunchedFromBackground : Message
```

Comment:
Incorrect naming (launch != enter), by logic this message should be sent when the app has entered the foreground from the background.